cfep-xx.md

```diff
@@ -0,0 +1,80 @@
+
+<table>
+<tr><td> Title </td><td> Make all Rust packages auditable </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Uwe L. Korn &lt;uwelk@xhochy.com&gt; </td></tr>
+<tr><td> Created </td><td> Jun 23, 2025</td></tr>
+<tr><td> Updated </td><td> Jun 23, 2025</td></tr>
+<tr><td> Discussion </td><td> N/A </td></tr>
+<tr><td> Implementation </td><td> <a href="https://github.com/conda-forge/rust-activation-feedstock/pull/79">rust-activation-feedstock#79</a> </td></tr>
+</table>
+
+## Abstract
+
+This proposal suggests integrating [`cargo‑auditable`](https://github.com/rust-secure-code/cargo-auditable) into the build process for all Rust-based feedstocks on conda‑forge.
+By enabling dependency‑tree embedding within compiled binaries, this integration would enhance downstream vulnerability auditing and transparency.
+
+## Motivation
+
+`cargo-auditable` embeds dependency information as a JSON blob into a dedicated linker section with [minimal overhead (\~4 kB), even for large dependency graphs (>400 crates)](https://github.com/rust-secure-code/cargo-auditable?tab=readme-ov-file#doesnt-this-bloat-my-binary).
+This enables downstream tooling (e.g. `cargo audit bin`, Trivy, Syft) to perform highly accurate vulnerability scans.
+
+This brings the Rust binaries on conda-forge en par with the Go binaries that have this enabled by default (see the output of `go version -m <binary>`).
+Multiple distributions (Alpine, NixOS, openSUSE, Void, Chimera) already adopt `cargo-auditable` in Rust package builds (see [Adoption](https://github.com/rust-secure-code/cargo-auditable?tab=readme-ov-file#adoption))
+
+## Proposal
+
+### 1. Add `cargo-auditable` as a hard dependency in `rust-activation`
+
+As we want to use this in all Rust builds, it should come as a dependency of the activation.
+Furthermore, we should set there the environment variable `CARGO` to `cargo auditable`.
+
+Implementation: [rust-activation-feedstock#79](https://github.com/conda-forge/rust-activation-feedstock/pull/79)
+
+### 2. Builds should use `$CARGO` instead of `cargo`
+
+Build scripts should be adapted to use `$CARGO` (or `%CARGO%`) instead of calls to plain `cargo`.
+While we could replace these calls also with calls to `cargo auditable`, this should serve as a more long-term solution if we want to supply more options.
+
+The usage of the `CARGO` environment variable also ensures that Python build using `maturin` also pick up `cargo-auditable` automatically (see [maturin#495](https://github.com/PyO3/maturin/issues/2495)).
+
+### 3. Add a lint to conda-smithy for `CARGO` usage
```

Comment:
This section should be about ensuring existing feedstocks migrate to the new setup. Recommended approaches can be done separately.